frontend/src/app/pages/about/template/template.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Location as NgLocation } from '@angular/common';
+import { HttpClient } from '@angular/common/http';
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Subject, takeUntil } from 'rxjs';
+
+@Component({
+    selector: 'app-template',
+    templateUrl: './template.component.html',
+    styleUrls: ['./template.component.scss'],
+})
+export class TemplateComponent implements OnDestroy, OnInit {
+    private readonly destroy$ = new Subject<void>();
+    content?: string;
+
+    /**
+     * The title of the page shown as h2 headline.
+     */
+    @Input() pageTitle = '';
+
+    /**
+     * The filename in the assets/about/ directory where the page content should be loaded from.
+     */
+    @Input() contentFile = '';
+
+    constructor(
+        private readonly location: NgLocation,
+        private readonly http: HttpClient
+    ) {}
+
+    ngOnInit(): void {
+        this.http
+            .get(`assets/about/${this.contentFile}`, { responseType: 'text' })
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((response) => (this.content = response));
```

Comment:
You could 
* refactor `content` to `content$?: Observable<string>`
```suggestion
        this.content$ = this.http
            .get(`assets/about/${this.contentFile}`, { responseType: 'text' });
```
* In the template use `content$ | async` instead of `content`
*  Remove `destroy$`

Reply:
Nice catch! This was still from the time when I set the placeholder texts as initial value in the TS file instead of the HTML template. Fixed it in https://github.com/hpi-sam/digital-fuesim-manv/pull/643/commits/88d0fa7ae6c2b99b62cf950b41c3f7dccb3af22d